pisek/tests/cms_test_suite.py

```diff
@@ -56,6 +56,10 @@ def runTest(self):
         if self.generator.cache_used:
             message = "\n  Generátor se nezměnil, používám vstupy vygenerované v předchozím běhu."
             print(termcolor.colored(message, color="cyan"))
+        else:
+            # Remove old solution outputs. The invalidated cache would not be used either
+            # way, but this way we get rid of possible junk
+            util.clean_data_dir(self.task_config, leave_inputs=True)
```

Comment:
Chceme `leave_inputs`? Teďka když se rozhodnu, že změním generátor, aby nic negeneroval, tak ho Písek vesele přeloží a spustí a zbydou tam vstupy z minule. To mi přijde, že fakt nechceme (takhle byla AFAIK rozbitá jedna úloha, staré testy, které v novém zadání nebyly validní), takže bych to případně přidal i do self testů.

Reply:
S tímhle byl problém, že tahle kontrola probíhá až po tom, co generátor doběhl, a pro CMS moc nevíme, jaké soubory generátor vytvořil. Vyřeším to jinak.